consensus/polybft/signer/public_test.go

```diff
@@ -0,0 +1,25 @@
+package bls
```

Comment:
This test file doesn't cover the majority of the functionality added in `public.go`

Reply:
Fixed through #825